src/opendap/auth/UrsIdP.java

```diff
@@ -120,7 +139,20 @@ private Element getConfigElement(Element config, String childName) throws Config
         return e;
     }
 
-
+    /**
+     * Another little worker method.
+     * @param config
+     * @param childName
+     * @return element found in `config`; will be `null` if `childName` not present
+     */
+    private Element getOptionalConfigElement(Element config, String childName) {
+        Element e = config.getChild(childName);
+        if(e == null){
+            String msg = this.getClass().getSimpleName() + " configuration does not contain optional <" + childName + "> element.";
+            log.error("{}",msg);
```

Comment:
Please consider triaging the log levels.  I would say that at worst, this is a `warn` or possibly an `info` log event.